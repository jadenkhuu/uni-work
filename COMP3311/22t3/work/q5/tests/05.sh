python3 ./q5 Rosehill 2019-08-10
