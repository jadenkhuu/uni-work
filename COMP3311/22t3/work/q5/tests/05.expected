Race meeting at Rosehill on 2019-08-10

Race 1, prize pool $5000, run over 1100m
Hart ridden by Willy Kan wins $3500
Tracery ridden by Jimmy Quinn wins $1000
Sandwich ridden by Rosemary Homeister Jr. wins $500

Race 2, prize pool $5000, run over 1500m
Spearmint ridden by Alice Van-Springsteen wins $3500
Great Northern ridden by Paddy Brennan wins $1000
Prince Palatine ridden by Johnny Loftus wins $500

Race 3, prize pool $1000, run over 1800m
Equine Intervention ridden by Tyler Baze wins $700
funny horses ridden by Ray Cochrane wins $200
Captain Cuttle ridden by Hayley Turner wins $100

Race 4, prize pool $3000, run over 2000m
Junkinthetrunk ridden by Yoshitomi Shibata wins $2100
Wheres the Beef ridden by Charlie Cunningham wins $600
Fifty Bales of Hay ridden by Bob Champion wins $300

Missile Stakes, prize pool $200000, run over 1200m
Dâ€™Apples Are Sweet ridden by Cash Asmussen wins $140000
Richard Friction ridden by William Buick wins $40000
Flying Scotsman ridden by Manuel Ycaza wins $20000
